tests/testthat/test-parsers.R

```diff
@@ -0,0 +1,31 @@
+context("parsers")
+
+test_that("parse_errors_to_df parses correct assertion type", {
+  data <- data.frame(
+    V1 = c(1, 0, -3)
+  )
+  validation <- validate(data) %>%
+    validate_if(V1 < 0)
+
+  expect_equal(parse_errors_to_df(validation)$type, error_id)
```

Comment:
I found this `error_id` quite confusing. It is not defined anywhere in the test, and it looks like dependency injection from another tests. When I run the tests locally it was not able to find this element. 

I initially commented this as a bug, but was surprised that `devtools::test` is passing. I found that `error_id` is a constant coming from the package script itself.
Still, I think it would be useful to comment that to make it more clear for anyone reading the tests only.

Reply:
thanks @mdubel. Comment added in [f689d26](https://github.com/Appsilon/data.validator/pull/59/commits/f689d269363c4a95e75583fcfce203b9a9a152d9).